components/utilities/object.js

```diff
@@ -0,0 +1,21 @@
+(function(d3, fc) {
```

Comment:
Any reason not to use [Object.Keys](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/keys)?